open_telemetry/README.md

```diff
@@ -0,0 +1,60 @@
+# OpenTelemetry Sample
+
+This sample shows how to configure OpenTelemetry to capture workflow traces and SDK metrics.
+
+For this sample, the optional `open_telemetry` dependency group must be included. To include, run:
+
+    poetry install --with open_telemetry
+
+To run, first see [README.md](../README.md) for prerequisites. Then run the following to start a Jaeger container to
+view the trace results:
+
+    docker run -d --name jaeger \
```

Comment:
does a user need to install this container separately?

Reply:
No, the container is automatically downloaded if not present by `docker`